beacon-light-client/plonky2/crates/circuits/src/deposit_accumulator_balance_aggregator_diva/final_layer.rs

```diff
@@ -256,7 +257,7 @@ fn hash_public_inputs<F: RichField + Extendable<D>, const D: usize>(
     balance_aggregation_pis: &CircuitOutputTarget<
         DepositAccumulatorBalanceAggregatorDivaFirstLevel,
     >,
-    deposits_commitment_mapper_pis: &CircuitOutputTarget<DepositsCommitmentMapperFirstLevel>,
+    deposits_commitment_mapper_pis: &CircuitOutputTarget<PubkeyCommitmentMapperFL>,
```

Comment:
Unify the naming. The proof is called `pubkey_commitment_mapper_proof`. You could also call it a root_proof